Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -406,10 +407,12 @@ extension TabScrollingController: UIScrollViewDelegate {
     }
 
     func scrollViewWillBeginZooming(_ scrollView: UIScrollView, with view: UIView?) {
+        configureRefreshControl(isEnabled: false)
         self.isUserZoom = true
     }
 
     func scrollViewDidEndZooming(_ scrollView: UIScrollView, with view: UIView?, atScale scale: CGFloat) {
+        configureRefreshControl(isEnabled: true)
```

Comment:
We should take the feature flag into account when re-enabling the refresh control.